### DISCUSSION TITLE
Disable build when fetching from git

### DISCUSSION THREAD

**Discussion by donferi** [yarnpkg/berry#2808] [Q&A]
Is there a way to disable the packaging step when fetching a package from git?

At my company we commit the `/dist` files and would like yarn to clone the repo but not run the packaging step. Some of our packages are failing when trying to build because of different node/npm versions, etc. Plus it has to pull down all the `devDependencies` of git packages which have more git references which yarn also tries to build.

**Comment by arcanis** [comment]
Nope, not out of the box. However you can refer to your package using a `.tar.gz` url, which will download the package without building it. GitHub offers automated download links, so you can probably use this trick to get a similar behavior.

**Reply by donferi** [reply]
@arcanis hey thanks for the quick reply. I think that would work fine for public repos, but not for private right?
